src/hosted.js

```diff
@@ -131,12 +133,15 @@
     }
 
     // Support loading from src/ or dist/ to make local dev easier
-    require(["bramble/client/main"], function(Bramble) {
+    if(window.location.pathname === "/dist/hosted.html") {
```

Comment:
I believe it's `<domain>/bramble/<staging|production>/dist/index.html`
where `<domain>` is `mozillathimblelivepreview.net` and it's either `staging` or `production` in the path.